aws/resource_aws_acm_certificate.go

```diff
@@ -63,23 +82,32 @@ func resourceAwsAcmCertificate() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
+						"validation_domain": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"validation_method": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"validation_emails": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem:     &schema.Schema{Type: schema.TypeString},
+						},
 					},
 				},
 			},
-			"validation_emails": {
```

Comment:
This is a breaking change for anyone who uses this attribute for outputs, etc. Please keep it and mark it with `Deprecated: "friendly message here"` if we really are going to remove it. It would also be helpful to know why this is happening in this PR.

Reply:
I have added deprecated for this attribute. it was moved to the certificate details attribute, as is more appropriate and consistent with the actual aws resource